src/views/Shapes/RoundedRectangle.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Modifiers } from "../../utils/modifiers";
+import { Rectangle } from "./Rectangle";
+
+type RoundedRectangleProps = Omit<Modifiers,"backgroundColor"> & {
+    fill: string
+};
+
+export const RoundedRectangle: React.FC<RoundedRectangleProps> = ({
```

Comment:
this component and the others can be reduced quite a bit like this:
```tsx
export const Capsule: React.FC<CapsuleProps> = ({
  cornerRadius = "50%",
  frame = { width: "100%", height: "100%" },
  ...rest
}) => {
  return <Rectangle cornerRadius={cornerRadius} frame={frame} {...rest} />;
};
```

Reply:
I like the cornerRadius solution, would we change the cornerRadius modifier to accept `string | number`. Right now is only `number`.

Reply:
Fixed in [8c172e0](https://github.com/andrew-levy/swiftui-react-native/pull/10/commits/8c172e0af1b8b252edf1857088c13ab4c7a7fe59)

Reply:
basically just forward all the props except for the ones we want to control